scripts/ci-e2e.sh

```diff
@@ -126,7 +123,7 @@ main(){
         HEART_BEAT_PID=$(echo $!)
     fi
 
-    if [[ "${E2E_FLAVOR}" == "powervs" ]]; then
+    if [[ "${E2E_FLAVOR}" == "powervs" || "${E2E_FLAVOR}" == "md-remediation" ]]; then
```

Comment:
`md-remediation` where is this flavour present?

Reply:
This flavour is present as a template here - https://github.com/kubernetes-sigs/cluster-api-provider-ibmcloud/tree/main/test/e2e/data/templates

Reply:
As I mentioned earlier for the long run we need to find a way to patch the existing template instead of having a new one.

Reply:
Will be raising a follow-up PR for it.